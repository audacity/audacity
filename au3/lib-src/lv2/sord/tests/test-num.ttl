@prefix eg: <http://example.org/eg#> .

eg:thing eg:num 0.1 .
eg:thing eg:num +0.2 .
eg:thing eg:num -0.3 .
eg:thing eg:num .4 .
eg:thing eg:num +.5 .
eg:thing eg:num -.6 .
eg:thing eg:num 1.58490e-05 .
eg:thing eg:num 1.58490e+05 .
eg:thing eg:num 1.58490e05 .
eg:thing eg:num 1.58490E05 .
