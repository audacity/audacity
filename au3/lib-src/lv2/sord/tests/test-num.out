<http://example.org/eg#thing> <http://example.org/eg#num> "0.1"^^<http://www.w3.org/2001/XMLSchema#decimal> .
<http://example.org/eg#thing> <http://example.org/eg#num> "+0.2"^^<http://www.w3.org/2001/XMLSchema#decimal> .
<http://example.org/eg#thing> <http://example.org/eg#num> "-0.3"^^<http://www.w3.org/2001/XMLSchema#decimal> .
<http://example.org/eg#thing> <http://example.org/eg#num> ".4"^^<http://www.w3.org/2001/XMLSchema#decimal> .
<http://example.org/eg#thing> <http://example.org/eg#num> "+.5"^^<http://www.w3.org/2001/XMLSchema#decimal> .
<http://example.org/eg#thing> <http://example.org/eg#num> "-.6"^^<http://www.w3.org/2001/XMLSchema#decimal> .
<http://example.org/eg#thing> <http://example.org/eg#num> "1.58490e-05"^^<http://www.w3.org/2001/XMLSchema#double> .
<http://example.org/eg#thing> <http://example.org/eg#num> "1.58490e+05"^^<http://www.w3.org/2001/XMLSchema#double> .
<http://example.org/eg#thing> <http://example.org/eg#num> "1.58490e05"^^<http://www.w3.org/2001/XMLSchema#double> .
<http://example.org/eg#thing> <http://example.org/eg#num> "1.58490E05"^^<http://www.w3.org/2001/XMLSchema#double> .
